lib/api/v3/attachments/attachment_metadata_representer.rb

```diff
@@ -0,0 +1,47 @@
+#-- encoding: UTF-8
+#-- copyright
+# OpenProject is a project management system.
+# Copyright (C) 2012-2015 the OpenProject Foundation (OPF)
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License version 3.
+#
+# OpenProject is a fork of ChiliProject, which is a fork of Redmine. The copyright follows:
+# Copyright (C) 2006-2013 Jean-Philippe Lang
+# Copyright (C) 2010-2013 the ChiliProject Team
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See doc/COPYRIGHT.rdoc for more details.
+#++
+
+require 'roar/decorator'
+require 'roar/json/hal'
+
+module API
+  module V3
+    module Attachments
+      class AttachmentMetadataRepresenter < ::API::Decorators::Single
+        property :file_name
+        property :description,
+                 getter: -> (*) {
+                   ::API::Decorators::Formattable.new(description, format: 'plain')
+                 },
+                 setter: -> (value, *) { represented.description = value['raw'] },
+                 render_nil: true
+      end
+    end
+  end
+end
```

Comment:
Sorry to pester you because I really appreciate you moving the stuff from the controller to the representer but this one here lacks specs.